storage/block/how-to/mount-and-use-volume.mdx

```diff
@@ -54,45 +59,62 @@ You can see your block volume named `sda` with the chosen storage size.
 
 ## How to format the Block Volume
 
-If you do not know which file system to use, you probably want to use `ext4`.
-```
-root@scw-festive-agnesi:~# mkfs.ext4 /dev/sda
-```
+1. Create a file system with the following command. This command uses the `ext4` filesystem, though you can choose another if you prefer.
 
-Make sure your file system is correctly created by running the `lsblk -f` command.
-```
-root@scw-festive-agnesi:~# lsblk -f
-NAME    FSTYPE LABEL UUID                                 MOUNTPOINT
-sda     ext4         925c2c17-ca8c-493a-a9cd-b475d87fd276
-vda
-├─vda1  ext4   root  8509fe04-d91c-410a-9087-c5d34537d3ae /
-└─vda15 vfat         B3E7-7040                            /boot/efi
-```
-Check if the `FSTYPE` field matches `ext4` for your Block volume.
+  ```
+  mkfs.ext4 /dev/sda
+  ```
+
+2. Make sure your file system is correctly created by running the `lsblk -f` command.
+
+  ```
+  lsblk -f
+  ```
+
+  You should see an output like the following. Check that the `FSTYPE` field matches `ext4` for your Block volume.
+
+  ```sh no-copy
+  root@scw-festive-agnesi:~# lsblk -f
+  NAME    FSTYPE LABEL UUID                                 MOUNTPOINT
+  sda     ext4         925c2c17-ca8c-493a-a9cd-b475d87fd276
+  vda
+  ├─vda1  ext4   root  8509fe04-d91c-410a-9087-c5d34537d3ae /
+  └─vda15 vfat         B3E7-7040                            /boot/efi
+  ```
 
 ## How to mount the Block Volume
 
 Once your file system is created, you need to mount it.  
 
 1. Create the mount point. In this tutorial, we choose `/mnt/block-volume`.
+
+  ```
+  mkdir /mnt/block-volume
   ```
-  root@scw-festive-agnesi:~# mkdir /mnt/block-volume
+
+2. Mount the volume. It is recommended to use the `defaults` option, as in the command below.
```

Comment:
I know we're supposed to avoid using "we", but we're also supposed to avoid passive voice, so I'll let you decide which one is best hehe
```suggestion
2. Mount the volume. We recommend you use the `defaults` option, as in the command below.
```